README.md

```diff
@@ -31,7 +31,7 @@ yarn add react-arborist
 ```
 
 ```
-npm install react-arborist
+npm install --save react-arborist
```

Comment:
Remove this change and we can merge this.

Reply:
Done